dbhydra/dbhydra_core.py

```diff
@@ -636,11 +672,13 @@ def _adjust_df(self, df: pd.DataFrame, debug_mode=False) -> pd.DataFrame:
 
         return df_copy
 
+
     def insert_from_df(self,df,batch=1,try_mode=False, debug_mode=False, adjust_df=False):
+
         if adjust_df:
             df = self._adjust_df(df, debug_mode)
 
-        assert len(df.columns)+1==len(self.columns) #+1 because of id column
+        # assert len(df.columns)+1==len(self.columns) #+1 because of id column
```

Comment:
what is the reason for this? I know it is problematic because it requires 1 (and only 1) id, but IMO if we delete this assert, then we need to solve cases with multiple id columns everywhere